services/api/src/middleware.py

```diff
@@ -79,6 +112,28 @@ def check_auth_exempt(method, path):
     return False
 
 
+def evaluate_tag(tag):
+    if not ENABLE_RSU_FEATURES and tag == "rsu":
+        return True
+    elif not ENABLE_INTERSECTION_FEATURES and tag == "intersection":
+        return True
+    elif not ENABLE_WZDX_FEATURES and tag == "wzdx":
+        return True
+    return False
+
+
+def is_feature_disabled(method, path):
+    if feature_tags.get(path) is not None:
+        if type(feature_tags.get(path)) is dict:
```

Comment:
Am I misunderstanding this or is this checking if a tag is a dictionary? We don't seem to have a case for this in the current list of feature_tags.

Reply:
I want to support restrictions on different methods within a path/endpoint. I added some descriptions and an example